src/components/Aggregator/AggregatorValues.js

```diff
@@ -20,6 +20,47 @@ export default class AggregatorValues extends Component {
     this.renderElement = props.renderElement || this._renderElement;
   }
 
+  buildAggregations = (fieldName, values, userSelection) => {
+    const parentNestedField = values.hasNestedField;
+
+    return Object.keys(values).map((key, index) => {
+      if (typeof values[key] === 'object') {
+        const aggrValue = values[key];
+        const checked = this.isItemChecked(fieldName, aggrValue, userSelection);
+        const _onUserSelectionChange = (e, { value }) => {
+          const aggrNameValue = {};
+          const aggTypeName = parentNestedField ? parentNestedField: fieldName; // assuming the invenio bug where sub-aggregations with the same name have one type of url request
```

Comment:
You cannot assume which backend you are using, the UI doesn't know it :)
I would re-phrase to something like:
`assuming that root fields names are unique`?

Reply:
I should've removed the line, it's not used anymore, now the keys are unique so it's not needed.